pandas/tests/io/formats/test_format.py

```diff
@@ -343,6 +343,16 @@ def test_repr_truncates_terminal_size(self):
 
         assert df2.columns[0] in result.split('\n')[0]
 
```

Comment:
nitpick: we have a pytest fixture formock now, that does the try / except / skip done above.

I think it'd be cleanest to split this into a new test here, and accept the `mock` parameter.

```python
def test_repr_truncates_terminal_size_full(self, mock):
    ...
```

Any if you're feeling adventurous, you could change the try / except / skip mock import above to use the fixture as well. Not a big deal though.